### DISCUSSION TITLE
Screen recoding optimization

### DISCUSSION THREAD

**Discussion by tao363** [FreeRDP/FreeRDP#10396] [Ideas]
When executing as a proxy, record the screen only when the local client is not minimized, otherwise no screen recording is performed

**Comment by akallabeth** [comment]
1. if your client disables screen updates during minimize there is nothing to be recorded. (so nothing to change on the proxy)
2. the proxy does not know the window state of your client and there is no way (in RDP) to get that. 
3. it the previous thing would be possible: the proxy recording would be quite flawed if the client can fake something and the proxy just stops recording data. (I´d just write a client that always tells it is minimized if I´d like to avoid session recording)
